python-checks/src/main/java/org/sonar/python/checks/DuplicatedMethodImplementationCheck.java

```diff
@@ -83,6 +93,34 @@ private static boolean isOnASingleLine(StatementList statementList, boolean hasD
     return statementList.statements().get(first).firstToken().line() == statementList.statements().get(statementList.statements().size() - 1).lastToken().line();
   }
 
+  private static void addQuickFix(IssueWithQuickFix issue, FunctionDef originalMethod, FunctionDef suspiciousMethod) {
+    ParameterList parameters = originalMethod.parameters();
+    if (parameters != null) {
+      List<AnyParameter> all = parameters.all();
+      if (all.size() == 1 && !ALLOWED_FIRST_ARG_NAMES.contains(all.get(0).firstToken().value())) {
+        return;
+      }
+      if (all.size() > 1) {
+        return;
+      }
+    }
+    boolean containsReturnStatement = originalMethod.body().statements().stream()
+      .anyMatch(s -> s.is(Tree.Kind.RETURN_STMT));
+    String replacementText = "";
+    if (containsReturnStatement) {
+      replacementText = replacementText + "return ";
+    }
+    replacementText = replacementText + originalMethod.name().name() + "()";
```

Comment:
You can not call a method like this. You have to call the method on a receiver. In case of a non-static method, you should use the self parameter `self.foo()`. In case of a class method you can do a static call like `Bar.foo()`.